src/main/webapp/dev/js/common/questionMcq.js

```diff
@@ -25,7 +28,23 @@ function addMcqOption(questionNum) {
             </span>
         </div>
     </div>
-    `).insertBefore($(`#mcqAddOptionRow-${questionNum}`));
+    `);
+
+    const weight = (`
+    <div class="margin-bottom-7px mcqChoiceRow-${curNumberOfChoiceCreated}-${questionNum}">
+        <input type="number" class="form-control nonDestructive" value="0"
+                id="${ParamsNames.FEEDBACK_QUESTION_MCQ_WEIGHT}-${curNumberOfChoiceCreated}-${questionNum}"
+                name="${ParamsNames.FEEDBACK_QUESTION_MCQ_WEIGHT}-${curNumberOfChoiceCreated}" step="0.01" required>
+    </div>
+    `);
+
+    if (curNumberOfChoiceCreated === 0) {
+        $(`#mcqChoices-${questionNum}`).html(choice);
+        $(`#mcqWeights-${questionNum}`).html(weight);
+    } else {
+        $(choice).insertAfter(lastChoice);
+        $(weight).insertAfter(lastWeight);
```

Comment:
How about assigning `$(`#mcqChoices-${questionNum}`)` to a variable and use `append` here?

Reply:
Done :+1: